src/transaction/transaction.rs

```diff
@@ -856,13 +944,11 @@ impl Committer {
                 }
             }
         };
-        self.bg_worker
-            .clone()
-            .spawn_ok(self.commit_secondary(commit_ts.clone()).map(|res| {
-                if let Err(e) = res {
-                    warn!("Failed to commit secondary keys: {}", e);
-                }
-            }));
+        tokio::spawn(self.commit_secondary(commit_ts.clone()).map(|res| {
```

Comment:
@andylokandy Then we will require the tokio runtime? Others LGTM.

Reply:
Yes

Reply:
timer and rwlock can be replaced with runtime independent ones.